### ISSUE TITLE
user.js breaks some extensions (apparently ToggleButton)

### ISSUE THREAD

**Issue by pyllyukko** [pyllyukko/user.js#12]
It seems that some setting in this user.js breaks some extensions. Apparently it affects the [ToggleButton](https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/ui_button_toggle) UI feature.

One~~Two~~ extension~~s~~ that is~~are~~ lacking the button~~s are~~ is ~~[Privacy Badger](https://www.eff.org/privacybadger) and~~ [Shodan Firefox Add-on](https://addons.mozilla.org/en-US/firefox/addon/shodan-firefox-addon/?src=api).

Some error relating to this issue:

```
*************************
A coding exception was thrown and uncaught in a Task.

Full message: TypeError: this.Paths is null
Full stack: Agent.wipe@resource:///modules/sessionstore/SessionWorker.js:236:7
worker.dispatch@resource:///modules/sessionstore/SessionWorker.js:21:24
anonymous/AbstractWorker.prototype.handleMessage@resource://gre/modules/workers/PromiseWorker.js:122:16
@resource:///modules/sessionstore/SessionWorker.js:30:41
```

**Comment by pyllyukko** [comment]
So setting ```browser.privatebrowsing.autostart == false``` brought the Shodan button back?!?

**Comment by pyllyukko** [comment]
Shodan add-on is still broken :/ Linking to #100.

**Comment by nodiscc** [comment]
I think this can be closed, it's tracked at #100
